lib/src/Base/Algo/openturns/ApproximationAlgorithmImplementation.hxx

```diff
@@ -104,6 +104,9 @@ public:
   virtual Collection<Indices> getSelectionHistory(Collection<Point> & coefficientsHistory) const;
   virtual Point getErrorHistory() const;
 
+  /** isModelSelection accessor */
+  virtual Bool isModelSelection() const;
```

Comment:
Comment from @jschueller : 

the naming feels weird as its not an adjective; "isModelSelected" or just "modelSelection" ?

Reply:
- `isModelSelected` : why not.
- `modelSelection`: aren't we supposed to name a method which returns a `bool` as "is*"?

What is your favorite option @regislebrun ?

Reply:
I vote for "involvesModelSelection" : this is the question answered according to your documentation.